src/utilities/include/vec_dh.h

```diff
@@ -226,9 +227,19 @@ class ManagedVec {
 
   const T &back() const { return *(ptr_ + size_ - 1); }
 
-  T &operator[](size_t i) { return *(ptr_ + i); }
+  T &operator[](size_t i) {
+    // ASSERT(i < size_, logicErr,
```

Comment:
This makes it quicker to find seg-fault locations but in `MANIFOLD_DEBUG` mode it makes the tests take twice as long, so I'm leaving it commented so we can easily use it just if we need to look for such things. The address sanitizer is another good way.